examples/js/loaders/FBXLoader.js

```diff
@@ -1989,7 +1989,7 @@
 
 			var rotation = modelNode.Lcl_Rotation.value.map( THREE.Math.degToRad );
 			rotation.push( 'ZYX' );
-			model.rotation.fromArray( rotation );
+			model.quaternion.setFromEuler( new THREE.Euler().fromArray( rotation ) );
```

Comment:
@takahirox @looeee I think `new` is OK here, too.

Reply:
Yep, that looks fine 👍